### PR TITLE
Added functions for model as required by fastai

### PR DESCRIPTION
**Pull request by jaredmejia**

### CODE REVIEW

File: Automator/AutoRand.py [hunk 1]
```diff
@@ -19,22 +19,21 @@
 # NOISE CONTROL
 
 # the standard deviation of the Gaussian that random angles are drawn from
-rand_angle_scale = pi / 36  # 5 degree s.d.
```
Comment (anthonyjclark) [inline Automator/AutoRand.py]: Is your autoformatter doing this?

File: Automator/AutoRand.py [hunk 2]
```diff
@@ -19,22 +19,21 @@
...
+rand_angle_scale = pi/36  # 5 degree s.d.
 
-# the minimum of the uniform distribution that random distances (to move) are drawn from
+# the minimum of the uniform distribution that random distances (to move) are drawn from
```
Comment (anthonyjclark) [inline Automator/AutoRand.py]: Why the extra space here?

**Review by anthonyjclark — COMMENTED** [review]
Looks like our code formatters are fighting. What are you using?

**Comment by jaredmejia** [comment]
The formatting issues should be fixed now

**Review by anthonyjclark — APPROVED** [review]

**Pull request merged**